framework/console/controllers/BaseMigrateController.php

```diff
@@ -465,11 +478,46 @@ public function actionCreate($name)
             throw new Exception("The migration name should contain letters, digits and/or underscore characters only.");
         }
 
-        $name = 'm' . gmdate('ymd_His') . '_' . $name;
-        $file = $this->migrationPath . DIRECTORY_SEPARATOR . $name . '.php';
+        $className = 'm' . gmdate('ymd_His') . '_' . $name;
+        $file = $this->migrationPath . DIRECTORY_SEPARATOR . $className . '.php';
 
         if ($this->confirm("Create new migration '$file'?")) {
-            $content = $this->renderFile(Yii::getAlias($this->templateFile), ['className' => $name]);
+            if (preg_match('/^CreateJoin(.+)And(.+)$/', $name, $matches)) {
```

Comment:
1º done!
2º rails use "CreateJoinTable" and laravel-generators use "make:migration:pivot", which do you suggest for yii? create_juntion_table_xxx_and_yyy where xxx and yyy are tables?